content/pages/company.html

```diff
@@ -2,7 +2,6 @@
   <head>
     <title></title>
     <meta name="template" content="page" />
-    <meta name="save_as" content="company.html" />
```

Comment:
I was also just considering dropping this page entirely, it seems to just overlap with our documentation.

This page needs some work though, it wasn't really ever finished.

Reply:
I think it's fine for now. It's only linked in the footer, and looks good.

Reply:
Aye, we have a lot of issues around it here that I've punted on. I might be more keen to just use the documentation version of this page instead of a duplicate page here. It's not a strong priority in either direction though.